src/runners/cairoRunner.ts

```diff
@@ -65,9 +65,22 @@ export class CairoRunner {
       throw new InvalidBuiltins(builtins, this.layout.builtins, layoutName);
 
     this.builtins = builtins;
-    const builtin_stack = builtins
-      .map(getBuiltin)
-      .map((builtin) => this.vm.memory.addSegment(builtin));
+    const builtin_stack = builtins.map((builtin) => {
+      const handler = getBuiltin(builtin);
+      if (builtin === 'segment_arena') {
+        const info = [
+          this.vm.memory.addSegment(handler),
+          new Felt(0n),
```

Comment:
what are these constants?

Reply:
The segment arena builtin initialization:
- base address of the info segment
- current number of dictionnaries (i.e. 0)
- current number of squashed dictionnaries (i.e. 0)

Reply:
renamed `info` to `initialValues` which is more meaningful